src/apps/invitations/errors.py

```diff
@@ -0,0 +1,9 @@
+from apps.shared.errors import ValidationError
+
+
+class AppletDoesNotExist(ValidationError):
+    pass
+
+
+class DoesNotHaveAccess(ValidationError):
+    pass
```

Comment:
I think that parents for your error classes are incorrect :(
Take a look `NotFoundError` and `PermissionError`